README.md

```diff
@@ -25,7 +25,19 @@ Example:
     >>> docrepr.sphinxify.rich_repr(oinfo)
     c:\users\user\appdata\local\temp\docrepr\tmpwvoj3s\rich_repr_output.html
 
+`docrepr` can also show a window with rendered documentation of a docstring
+if you have PyQt4 or PySide installed:
+
+    from docrepr import webview
+    s = set([1,2,3])
+    webview(s, oname='bar')
+
+`oname` is a just a name for the object, because Python loses it when
+variable is passed to function.
+
+![webview example](example.png)
```

Comment:
Please use as an example screenshot something meaningful, such as the docrepr output of `numpy.sin`